### PR TITLE
Issue 274 oeh logos

### LINKED ISSUE #274: Add supporter logos for austrian student body organizations
Please add the logos of the following institutions on the starting page as supporting orgs:

https://ohmedwien.sharepoint.com/:f:/s/Corona-Stab/Eq8s4_4q9DdLjlMNDKTuafoBWcR_2YM1nMWM8ogKAhT4lA?e=gRRMhv

@andyz02

### LINKED ISSUE #322: Rebranding
Make us an opensource team

- [x] branding auf der Website match4healthcare umändern zu "mit Unterstützung von OPen Source Developer Team..."
- [x] Github orga umbennen
- [ ] github readme entsprechend anpassen

### LINKED ISSUE #326: Map: Different color for facilities
The facility mapview should use a different color than the supporter mapview. Proposal: Blue instead of pink from our main theme colors.

### PR DESCRIPTION
**Pull request by maltezacharias**
Cherry picked from andreas 

Closes #322

### CODE REVIEW

**Review by bjrne — COMMENTED** [review]
Reviewed and tested locally until here, works fine

**Comment by josauder** [comment]
I think we should finish the discussion concerning the about page before someone can review

**Comment by feeds** [comment]
Please merge the change for margburger bund on staging

File: backend/apps/ineedstudent/templates/map_hospitals.html [hunk 1]
```diff
@@ -37,40 +110,38 @@
...
+</svg></h4> <p id="user_count" style="margin-bottom:0px; display:inline;"></p> Hilfesuchende</div>	
+	
+	 <a href="http://mapbox.com/about/maps" class="mapbox-wordmark" target="_blank">Mapbox</a>
+	
+</div>
+<script type="text/javascript">
+
+	var mymap = L.map('mapid').setView([51.13, 10.018], 6);
+   L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}@2x?access_token=pk.eyJ1Ijoiam9zYXVkZXIiLCJhIjoiY2s4MWNxa2QzMDNkejNrcDVhbzV6MDMxYSJ9.O4sLC7VSSgy4S7EKjNoG1g', {
+    attribution: ' <a href="https://www.mapbox.com/about/maps/">© Mapbox</a> | <a href="http://www.openstreetmap.org/copyright">© OpenStreetMap</a> | <a href="https://www.mapbox.com/map-feedback/" target="_blank">Improve this map</a>',
```
Comment (Baschdl) [inline backend/apps/ineedstudent/templates/map_hospitals.html:129]: @andyz02 Is that the new or old attribution?

**Comment by Baschdl** [comment]
Untested: maps

**Review by Baschdl — APPROVED** [review]
Will test maps on staging

**Pull request merged**